.ci/Jenkinsfile_security_cypress

```diff
@@ -0,0 +1,17 @@
+#!/bin/groovy
+
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+
+kibanaPipeline(timeoutMinutes: 180) {
+  catchError {
+    workers.base(size: 's', ramDisk: false) {
+      kibanaPipeline.bash('test/scripts/jenkins_security_solution_cypress.sh', 'Execute Security Solution Cypress Tests')
+    }
+  }
+
+  if (params.NOTIFY_ON_FAILURE) {
+    slackNotifications.onFailure(channel: '#security-solution-slack-testing')
```

Comment:
Can you move this up like in `master`? https://github.com/elastic/kibana/blob/master/Jenkinsfile#L7

I moved it recently to enable in-progress notifications